### ISSUE TITLE
EnlightenGAN does not converge 

### ISSUE THREAD

**Issue by gosnold** [wkhademi/ImageEnhancement#3]
Hi,

I tried to train enlightenGAN using python train/enlightengan_train.py --dirA /enlightenGAN/trainA --dirB /enlightenGAN/trainB --weight_init_gain 0.02 --scale_size 320 --crop_size 320 --patchD_3 5 --self_attention --times_residual --patchD --vgg --patch_vgg --use_ragan --hybrid_loss --batch_size 16  --niter 6000 --niter_decay 6000 --lr 0.0001

But the generator loss does not decrease:
![2021-03-10_16-59](https://user-images.githubusercontent.com/18752207/110658548-10c64580-81c2-11eb-93b3-6d4e3a180807.png)

And I got the similar results with initial learning rates of 1e-3 and 1e-5.
Is there something I am doing wrong? Did you successfully train it?

Thank for your consideration.
